app/views/grader_permissions/_grader_permission_pane.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class='pane-wrapper'>
+  <div class='pane'>
+    <div id='grader_permission_pane'>
+      <%= render partial: 'shared/navigation_warning',
+                 formats: [:js],
+                 handlers: [:erb] %>
+      <%= form_for @user, as: :user do |f| %>
+        <div class='inline-labels'>
+          <%= f.label :user_name, User.human_attribute_name(:user_name) %>
+          <%= f.text_field :user_name %>
+
+          <%= f.label :last_name, User.human_attribute_name(:last_name) %>
+          <%= f.text_field :last_name %>
+
+          <%= f.label :first_name, User.human_attribute_name(:first_name) %>
+          <%= f.text_field :first_name %>
+
+          <%= f.label :email, User.human_attribute_name(:email) %>
+          <%= f.text_field :email %>
+        </div>
+
+        <%= f.hidden_field :id %>
```

Comment:
is this necessary?

Reply:
That is only because the `update` method is looking for the id there: 

```ruby
  def update
    @user = Ta.find(params[:user][:id]) # <- this line
    @user.update(user_params)
    respond_with(@user)
  end
```
The id of the ta can be accessed with `params[:id]` which is passed to the controller either way. If you get the id through `params[:id]` the hidden field is not necessary.